src/components/org/OrgChart/components/Aside.tsx

```diff
@@ -62,24 +63,68 @@ function Aside<T>() {
     };
 
     const renderRow = useCallback(
-        (cards: OrgNode<T>[], rowNo: number) => {
+        (cards: OrgNode<T>[], rowNo: number, additionalRowHeight: number) => {
             const startX = getStartXPosition();
             return cards.map((card, i) => (
                 <Card
                     key={card.id}
                     node={card}
                     x={startX + i * (cardWidth + cardMargin)}
-                    y={(rowNo + 1) * (rowMargin - 20) + 24}
+                    y={(rowNo + 1) * (rowMargin - 20) + 24 + additionalRowHeight}
                 />
             ));
         },
         [centerX, cardWidth, cardMargin, rowMargin, numberOfCardsPerRow]
     );
 
+    const getAdditionalRowHeight = useCallback(
+        (cards: OrgNode<T>[]) => {
+            const greatestAdditionPersons = cards.reduce((highest: number, card: OrgNode<T>) => {
+                if (card.numberOfAssignees && card.numberOfAssignees > highest) {
+                    return card.numberOfAssignees;
+                }
+                return highest;
+            }, 0);
+            if (greatestAdditionPersons === 0) {
+                return 0;
+            }
+            return (
+                Math.ceil(greatestAdditionPersons / 3) *
+                (numberOfCardsPerRow === 1 ? rowMargin - 20 : rowMargin)
+            );
+        },
+        [numberOfCardsPerRow, rowMargin]
+    );
+
+    const additionalRowHeight = rows.reduce(
```

Comment:
This should probably be memoized?

Reply:
Memorized and moved common logic to a hook